README.md

```diff
@@ -146,6 +146,15 @@ WMCO is not responsible for Windows operating system updates. The cluster admini
 creating the VMs and hence, the cluster administrator is responsible for providing an updated image. The cluster 
 administrator can provide an updated image by changing the image in the MachineSet spec.
 
+## Enabled features
+
+### Autoscaling Windows nodes
```

Comment:
@EricPonvelle is there a way to call out that this feature will be enabled for Windows nodes in our 2.0.1 release? Do you release note it? Or add it to the [Cluster Autoscaling section](https://docs.openshift.com/container-platform/4.7/machine_management/applying-autoscaling.html)? Anything works for us as long as it is called out.

/cc @Anandnatraj